src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.NonNetFx.Windows.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    public static partial class PlatformDetection
+    {
+        public static bool TargetsNetFx452OrLower => false;
+        public static bool IsNetfx462OrNewer => false;
+        public static bool IsNetfx470OrNewer => false;
+        public static bool IsNetfx471OrNewer => false;
+        public static bool IsNetfx472OrNewer => false;
+
+        public static string LibcRelease => "glibc_not_found";
```

Comment:
These two should go into PlatformDetection.Windows.cs shouldn't they ? They are for all windows flavors.

Reply:
I suggest using the non empty string for all